BridgeAppSDK/SBAJSONObject.m

```diff
@@ -197,40 +185,40 @@ - (id)jsonObject {
 
 @end
 
-id sba_JSONObjectForObject(id object, NSString * key, NSDictionary <NSString *, NSFormatter *> *formatterMap) {
-    if ([object respondsToSelector:@selector(jsonObjectWithFormatterMap:)]) {
-        return [object jsonObjectWithFormatterMap:formatterMap];
+id SBAJSONObjectForObject(id object) {
+    if (![object isKindOfClass:[NSObject class]]){
+        // If this is not an object that inherits from NSObject class then return NSNull
+        return [NSNull null];
```

Comment:
I'd like to either have this assert rather than just silently returning NSNull, or have the method signature take an NSObject, as it indicates a programming error.